modules/tinymce/src/core/main/ts/fmt/RemoveFormat.ts

```diff
@@ -471,9 +471,14 @@ const remove = (ed: Editor, name: string, vars?: FormatVars, node?: Node | Range
   const isRemoveBookmarkNode = (node: Node) => Bookmarks.isBookmarkNode(node) && NodeType.isElement(node) && (node.id === '_start' || node.id === '_end');
 
   // Merges the styles for each node
-  const process = (node: Node) => {
+  const process = (nodes: Node[], node: Node) => {
```

Comment:
This can be reverted now, as `nodes` is no longer used.

- [ ] Revert back to just passing a single node

Reply:
updated